math/area.cpp

```diff
@@ -36,32 +54,40 @@ T square_area(T length) {
  * @param width is the width of the rectangle
  * @returns area of the rectangle
  */
-template <typename T>
-T rect_area(T length, T width) {
-    return length * width;
+uint64_t rect_area(uint32_t length, uint32_t width) {
+    /*
+        You may notice that these variables are prepended with (uint64_t)
+        This is because when doing the multiplication we need to give these data
+        types a wider area of memory when doing this calculation so we need a 64
+       bit variable.
+
```

Comment:
Yea, I wanted to keep it simple though because of the context of this issue: https://github.com/TheAlgorithms/C-Plus-Plus/issues/2456

I can update these to use floats if you guys want and add more checks. It will increase the complexity though.

Reply:
I think we need input from @Panquesito7 is this too simple? it will lack the functionality to do floats and doubles and decimals.